README.md

```diff
@@ -59,19 +57,51 @@ To use a standalone gunicorn server with 4 worker threads:
 ```
   gunicorn -w 4 -b 0.0.0.0:5001 event_app:app
 ```
+
+Setup Reporting APP:
+=====================
+configure and setup ensembl-prodinf-reporting (https://github.com/Ensembl/ensembl-prodinf-reporting) to update the event workflow status in Elastic search.
+
+Add [DEFAULT] section in .config.ini file (https://github.com/Ensembl/ensembl-prodinf-reporting/blob/main/.config.ini).
+
+```
+[DEFAULT]
+DEBUG=True
+REPORTER_TYPE=elasticsearch
+AMQP_HOST=localhost
```

Comment:
I would like to have same parameter set such as we do for handover app settings, so something more like a `uri` would be better. AMQP_HOST="user:pass@host:port".
Even better would be to use the same settings names as Handover, e.g. CELERY_HOST

Reply:
AMQP_HOST param is  related to reporting app (.config.ini). event app register to reporting app to publish the message to elasticsearch . This example is to step all depents app required for event app.